Snippets/Sqs/Sqs_All/QueueNameHelper.cs

```diff
@@ -29,8 +29,8 @@ public static string GetSqsQueueName(string destination, string queueNamePrefix
 
             if (s.Length > 80)
             {
-                throw new Exception(
-                    $"Address {destination} with configured prefix {queueNamePrefix} is longer than 80 characters and therefore cannot be used to create an SQS queue. Use a shorter queue name.");
+                throw new ArgumentException(
```

Comment:
These changes should be ported to the PowerShell version as well (currently copy pasted into each PS file)